### PR TITLE
update tictactoe contracts

### PR DESCRIPTION
**Pull request by lilyanB**

### CODE REVIEW

File: .github/workflows/tictactoe-daily-deploy.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+name: Tictactoe Daily Deploy
+
+on:
+  push:
+    paths:
+      - sum/**
```
Comment (julienbrs) [inline .github/workflows/tictactoe-daily-deploy.yml:6]: shouldn't it be `games?/**`

Comment (lilyanB) [reply .github/workflows/tictactoe-daily-deploy.yml]: oops, copy/paste error ;)
thanks

done

**Review by julienbrs — APPROVED** [review]
LGTM!

**Review by Ben-Rey — APPROVED** [review]

File: .github/workflows/tictactoe-daily-deploy.yml (hunk 1 above)
Comment (Ben-Rey) [reply .github/workflows/tictactoe-daily-deploy.yml]: you're right. @lilyanB Can you fix it in an other branch

**Pull request merged**